pages/add/add.jsx

```diff
@@ -254,7 +254,7 @@ class Add extends React.Component {
               clarity and style.
             </p>
             <button
```

Comment:
Since it's actually the error message that's shifting position, I would apply changes to it instead of to the `<button>`.

1. Remove `mr-3` from `<button>`
2. Wrap the error message(s) in a `<div>` and add appropriate classes to it
3. In `form.scss` around line 52, change the selector from `button + span` to `.error`

e.g., 

```jsx
<button
  className="btn btn-primary"
  type="submit"
  onClick={evt => this.handleFormSubmit(evt)}
  disabled={this.state.submitting ? `disabled` : null}
>
  {this.state.submitting ? SUBMITTING_LABEL : PRE_SUBMIT_LABEL}
</button>
<div className="d-sm-inline-block mt-2 mt-sm-0 ml-sm-3">
  {authErrorMessage}
  {serverErrorMessage}
  {this.state.showFormInvalidNotice && (
    <span>Something isn't right. Check your info above.</span>
  )}
</div>
```